functions/lib/sql-helper.js

```diff
@@ -26,3 +26,25 @@ module.exports.sqlFindCheckByCheckCode = async function (checkCode) {
   const res = await sqlService.query(sql, params)
   return R.head(res)
 }
+
+/**
+ * Retrieve all pupil checks based on check code
+ * @param checkCode
+ * @return {Promise<object>}
+ */
+module.exports.sqlFindChecksByCheckCode = async function (checkCode) {
+  const sql = `SELECT * from mtc.mtc_admin.[check]
```

Comment:
There is no need to explicitly state the database name, that's determined by someone else.